e2e-tests/scheduled-backup/conf/cr.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: pg.percona.com/v1
```

Comment:
I believe it's worth constructing, not keeping and constantly updating between releases

Reply:
https://github.com/percona/percona-postgresql-operator/pull/114/commits/df5ca42e0f4239a42855c9ba531d71cee3911a60